### ISSUE TITLE
Plans to submit PRs upstream?

### ISSUE THREAD

**Issue by flowchartsman** [qntfy/jsonparser#7]
I came here looking for a flexible JSON Parser/Munger, and I note that this repo is (as of this writing) 5 commits ahead and 45 behind upstream buger/jsonparser. Are there any plans to sync up, or is this repo different enough to merit a hard fork?

**Comment by JoshuaC215** [comment]
Hi @flowchartsman, we started working on this fork at a time when we needed the new features here and it seemed like it would take too long to get them merged into the upstream. IIRC there were some nice new additions in buger/jsonparser since then but nothing critical for our use and we haven't needed to make any updates here in quite a while.

I think we would welcome efforts from others to help sync up, but it's pretty low on the stack for us right now so unlikely that anyone on our team would make a pass at it any time in the next few months.

Certainly if anyone wanted to take any of the functions / functionality we've added here and separately integrate them into the upstream that would also be totally welcome :)

**Comment by flowchartsman** [comment]
Yep, understood, that's often the way of things. If it's still split when I get around to integrating, I'll take a crack at it!
